packages/fxa-settings/src/pages/Signin/en.ftl

```diff
@@ -0,0 +1,22 @@
+# Strings within the <span> elements appear as a subheading.
+signin-password-needed-header = Enter your password <span>for your { -product-firefox-account }</span>
+
+# $serviceLogo - an image of the logo of the service which the user is authenticating for.
+# For languages structured like English, the phrase can read "to continue to"
+signin-subheader-with-logo = Continue to <span>{ $serviceLogo }</span>
+
+# $serviceName - the name of the service which the user authenticating for
+# For languages structured like English, the phrase can read "to continue to { $serviceName }"
+signin-subheader-without-logo = Continue to { $serviceName }
+signin-button = Sign in
+signin-header = Sign in
+# <br /> is just spacing for visual display. <linkExternal> links to the Terms of Service and also to the Privacy Notice
+firefox-and-pocket-tos = By proceeding, you agree to: <br /> { product-pocket }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal><br />{ -brand-firefox}’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>
```

Comment:
nit: space between `<br/>` and variable in `<br /> { product-pcocket }` but not `<br />{ -brand-firefox }`. Whitespace is weird and might not affect front end UI, but possibly worth calling out.